bokehjs/src/coffee/models/glyphs/glyph.coffee

```diff
@@ -55,6 +55,18 @@ export class GlyphView extends BokehView
     bb = {minX: d.minX, minY: d.minY, maxX: d.maxX, maxY: d.maxY}
     return @_bounds(bb)
 
+  log_bounds: () ->
+    if not @index?
+      return bbox.empty()
+    log_x_index = rbush()
+    log_y_index = rbush()
+    positive_x_pts = @index.search(bbox.positive_x())
+    positive_y_pts = @index.search(bbox.positive_y())
+    log_x_index.load(positive_x_pts)
+    log_y_index.load(positive_y_pts)
+    bb = {minX: log_x_index.data.minX, minY: log_y_index.data.minY, maxX:log_x_index.data.maxX, maxY:log_y_index.data.maxY}
+    return bb
+
```

Comment:
The seed was planted when I was working with rbush to fix a bug with box select some time ago! I tried out the linear search and it's much faster than using a new index, so I'll change that. 

Another thing that I encountered is that the index `search` returns bboxes (points or rectangles) that are either contained in or intersect with the bbox passed in. So, searching the positive half plane will also return rectangle bboxes (Rect, Quad, etc.) that cover both the positive and negative side of 0.